Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -1111,6 +1121,19 @@ public static void RpcCall(string name, object data, RpcMode mode, Action callba
                 }
             }
 
+
+            JSONArray jsonArray = new JSONArray();
+            foreach (string item in rpcCalledEvents)
+            {
+                jsonArray.Add(item);
```

Comment:
Can this overflow, as you're not checking for duplicates or dead references?

Reply:
```c#
  if (!rpcCalledEvents.Contains(name))
  {
      rpcCalledEvents.Add(name);
  }
```

Duplicates aren't added to the list, this check if before the JSON serialization.